lib/package-managers/npm.js

```diff
@@ -85,6 +88,49 @@ function isPre(version) {
     return versionUtil.getPrecision(version) === 'release';
 }
 
+/**
+ * @param {{}}     versions    Object with all versions
+ * @param {String} enginesNode Package engines.node range
+ * @returns {Array} An array of versions which satisfies engines.node range
+ */
+function filterSatisfiesMinVersion(versions, enginesNode) {
```

Comment:
Let's call this function `doesSatisfyEnginesNode` to better capture the purpose.